packages/api/src/brevo-contact/brevo-contacts.service.ts

```diff
@@ -1,10 +1,20 @@
 import { Injectable } from "@nestjs/common";
 
-import { BrevoApiContactsService } from "../brevo-api/brevo-api-contact.service";
+import { BrevoApiContactsService, CreateDoubleOptInContactData} from "../brevo-api/brevo-api-contact.service";
+import { SubscribeResponse } from "./dto/subscribe-response.enum";
 
 @Injectable()
 export class BrevoContactsService {
     constructor(private readonly brevoContactsApiService: BrevoApiContactsService) {}
 
-    // TODO: add functionality back here when brevo contact/target list is implemented
+
+    public async createDoubleOptInContact(data: CreateDoubleOptInContactData, templateId: number): Promise<SubscribeResponse> {
+        const contactListId = 2;
```

Comment:
This id should also be configurable.

Reply:
Yes, this will be done when the contact lists/target lists will be added. I included a `TODO` comment for now in [8a87f90](https://github.com/vivid-planet/comet-brevo-module/pull/7/commits/8a87f90f443fe1f461afb7f431bb30b61e406bac)